docs/current/guides/snippets/use-secrets/host-env/index.mjs

```diff
@@ -0,0 +1,19 @@
+import { connect } from "@dagger.io/dagger"
+
+// initialize Dagger client
+connect(async (client) => {client
+  // read secret from host variable
+  const secret = client.host().envVariable("GH_SECRET").secret()
+
+  // use secret in container environment
+  const out = await client.
+    container().
+    from("alpine:3.17").
+    withSecretVariable("GITHUB_API_TOKEN", secret).
+    withExec(["apk", "add", "curl"]).
+    withExec(["sh", "-c", `curl "https://api.github.com/repos/dagger/dagger/issues" --header "Accept: application/vnd.github+json" --header "Authorization: Bearer $GITHUB_API_TOKEN"`]).
+    stdout()
+
+  // print result
+  console.log(out)
+}, {LogOutput: process.stderr})
```

Comment:
👍 Most examples in js use `stdout` in `LogOutput`. They should all use `stderr` (in all SDKs).